rule/import-naming.go

```diff
@@ -0,0 +1,82 @@
+package rule
+
+import (
+	"fmt"
+	"regexp"
+	"sync"
+
+	"github.com/mgechev/revive/lint"
+)
+
+// ImportNamingRule lints import naming.
+type ImportNamingRule struct {
+	configured       bool
+	namingRule       string
+	namingRuleRegexp *regexp.Regexp
+	sync.Mutex
+}
+
+const defaultNamingRule = "^[a-z][a-z0-9]$"
```

Comment:
```suggestion
const defaultNamingRule = "^[a-z][a-z0-9]*$"
```

Reply:
It was a mistake. It should have been:

"^[a-z][a-z0-9]{0,}$"

Adding `*` will mean only one symbol. I will update it.

Reply:
Applied according to my pattern.